feat: change repo structure and package name (#12)
MetaMask/snap-watch-only: README.md

# Account Watcher Snap

> This repository contains a keyring snap for watch-only accounts.

Reviewer: ```suggestion
This repository contains a snap to manage watch-only accounts.
```
